lint.go

```diff
@@ -1444,6 +1445,24 @@ func (f *file) lintContextArgs() {
 	})
 }
 
+// lintGoPragmas lints go:... pragmas
+func (f *file) lintGoPragmas() {
+	for _, cg := range f.f.Comments {
+		for _, c := range cg.List {
+			txt := c.Text
+			if !strings.HasPrefix(txt, "//") {
+				continue
+			}
+			txt = txt[2:]
+			// if there's space before go:, this was probably a mistake
+			if !strings.HasPrefix(txt, "go:") &&
+				strings.HasPrefix(strings.TrimSpace(txt), "go:") {
+				f.errorf(cg, 0.9, category("comments"), "go: pragmas should have no spaces between // and go:")
```

Comment:
"should" is a misnomer here. it implies that you _should_ but it's not required. I would make this more explicit.